apps/api/src/app/workflows-v2/mappers/workflow-template-get-mapper.ts

```diff
@@ -0,0 +1,80 @@
+import { IPreferenceChannels, StepTypeEnum } from '@novu/shared';
+import { DiscoverWorkflowOutputPreferences } from '@novu/framework';
+import { NotificationStepEntity, NotificationTemplateEntity } from '@novu/dal';
+
+import { MinifiedResponseWorkflowDto, StepDto, WorkflowResponseDto } from '../dto/workflow.dto';
+
+export class WorkflowTemplateGetMapper {
+  static toResponseWorkflowDto(template: NotificationTemplateEntity): WorkflowResponseDto {
+    return {
+      _id: template._id,
+      workflowId: template.name,
+      tags: template.tags,
+      active: template.active,
+      payload: template.data ? { schema: template.data } : undefined, // Assuming data is mapped to payload schema
```

Comment:
payload and data are 2 different entities.
`payload` is what you send while you trigger a notification.
`data` is the information you want to store in Novu to reuse in the notification, users used it in v1, but I'm not sure how useful it is in v2.